salt/modules/textfsm_mod.py

```diff
@@ -14,12 +14,12 @@
 .. note::
 
     For Python 2/3 compatibility, it is more recommended to
```

Comment:
Shouldn't we remove the Py2/3 comment too?

Reply:
Yes, probably should.

Reply:
Done